examples/eventstreamsadminv1/README.md

```diff
@@ -0,0 +1,17 @@
+# Event Streams Admin V1 API examples
+
+The Event Streams Admin V1 API allows to provision and manage topics.
+
+# Setting Partitions and PartitionCount
+
+If Partitions and PartitionCount are both not set they will be initialized to 1. If Partitions are set and PartitionCount is not set, PartitionCount will be initialized to what Partitions is. If Partitions is not set and PartitionCount is set then Partitions will be initialized to what PartitionCount is. 
+
```

Comment:
I would change `PartitionCount will be initialized to what Partitions is` to `PartitionCount will be initialized to the value set in Partitions` and `Partitions will be initialized to what PartitionCount is.` to `Partitions will be initialized to the value set in PartitionCount.`